build.gradle

```diff
@@ -5,9 +5,10 @@ buildscript {
     'targetSdk': 28,
     'sourceCompatibility': JavaVersion.VERSION_1_7,
     'targetCompatibility': JavaVersion.VERSION_1_7,
+    'androidx': '1.0.0-rc02',
+    'androidxLifecycle': '2.0.0-rc01',
```

Comment:
These should be inlined. It's only coincidence that they're all at the same version now. Once 1.0 lands they will be versioned individually like normal libraries.

Reply:
great!